### PR TITLE
Verify the new wtime and optime access log keywords

### PR DESCRIPTION
**Pull request by sgouvern**
Description:
In dirsrvtests/tests/suites/ds_logs/ds_logs_test.py:
 new test added: test_optime_and_wtime_keywords
 test_etime_order_of_magnitude adapted to the new RESULT string format in the access log

Relates: https://bugzilla.redhat.com/show_bug.cgi?id=1850275

Reviewed by: ??

### CODE REVIEW

File: dirsrvtests/tests/suites/ds_logs/ds_logs_test.py [hunk 1]
```diff
@@ -839,21 +840,103 @@ def test_etime_order_of_magnitude(topology_st, clean_access_logs, remove_users,
...
+         4. Search operation is successful
+         5. SRCH operation log string is catched
+         6. op number is collected
+         7. RESULT string is catched from the access log
+         8. wtime and optime keywords are collected
+         9. wtime, optime and etime values are collected
+         10. (optime + wtime) =~ etime
+    """
+
+    entry = DSLdapObject(topology_st.standalone, DEFAULT_SUFFIX)
```
Comment (droideck) [inline dirsrvtests/tests/suites/ds_logs/ds_logs_test.py]: Generally, you can use 'Domain' class for the root suffix but I don't see how 'entry' object is used in this code at all...

**Review by droideck — APPROVED** [review]
The rest looks good!

File: dirsrvtests/tests/suites/ds_logs/ds_logs_test.py (hunk 1 above)
Comment (sgouvern) [reply dirsrvtests/tests/suites/ds_logs/ds_logs_test.py]: Right, the entry object is not used. I did a small change to remove the line in both tests.
@droideck could you please review again ? Thanks !

**Comment by Firstyear** [comment]
All seems reasonable to me too :)

**Comment by sgouvern** [comment]
Thanks @Firstyear and @droideck for your reviews :) 
@droideck could you please have a final look and merge ? 
Thanks !

**Review by droideck — APPROVED** [review]
Everything still looks good! :)
Merging...

**Pull request merged**